.npmrc

```diff
@@ -1,2 +1,3 @@
 tag-version-prefix=""
 message="chore(release): %s"
+registry=https://registry.npmjs.org
```

Comment:
Is this intentionally part of this PR?

Reply:
Ah, this change isn't related to this PR, but I needed it so that yarn wouldn't try pull packages from another registry that I have configured.
I don't think anyone else would have this issue, but it wouldn't cause an issue leaving it in as well.
Happy to remove if you think so

Reply:
No problem with me, just asked since it seemed unrelated 😄